src/playground/components/Configuration.js

```diff
@@ -64,6 +64,15 @@ const customStyles = {
         color: "var(--headings-color)",
         cursor: "pointer",
         backgroundColor: "var(--lighter-background-color)"
+    }),
+    noOptionsMessage: styles => ({
+        ...styles,
+        backgroundColor: "var(--body-background-color)"
+    }),
+    menuList: styles => ({
+        ...styles,
+        backgroundColor: "var(--body-background-color)",
+        border: "1px solid var(--border-color)"
```

Comment:
Already there is a border for all options? Why are we introducing border again for the menuList?
| Before (main branch) | After  (your branch) | 
| ---- | ---- | 
| <img width="246" alt="Screenshot 2022-08-13 at 1 25 17 PM" src="https://user-images.githubusercontent.com/32865581/184474764-859d16dd-60d9-478e-96d3-941ce1143ed2.png"> | <img width="244" alt="Screenshot 2022-08-13 at 1 25 29 PM" src="https://user-images.githubusercontent.com/32865581/184474770-a683d0ac-da05-4bde-9f2e-977985d0bef9.png"> |

Reply:
Yes, we can remove the border here.

```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/42532987/184476778-0633ee47-bd76-499a-bd86-373615373e1d.png)
Without the border property in `menuList`, `No options` will appear with no border.

Will fix it.